### ISSUE TITLE
Include post tags as hashtags

### ISSUE THREAD

**Issue by Dilden** [Dilden/x-post#3]
If a post has tags attached to it, those should be included in social media posts as PascalCase for accessibility. 

Example: post has tags linux, home lab
social: #Linux #HomeLab

**Comment by Dilden** [comment]
Problem here is that posts are being sent to 3rd party channels via `transition_post_status` and tags have not necessarily been saved to the post yet in the database.

Using `publish_post` hook works as it fires after related taxonomies have been saved, but then anytime the post is updated it fires again.

A possible workaround for now is adding tags to post excerpt as that is included right away.
